addons/interact_menu/XEH_clientInit.sqf

```diff
@@ -12,13 +11,13 @@ GVAR(ParsedTextCached) = [];
 
 ["ace_settingChanged", {
     params ["_name"];
-    if (({_x == _name} count [QGVAR(colorTextMax), QGVAR(colorTextMin), QGVAR(colorShadowMax), QGVAR(colorShadowMin), QGVAR(textSize), QGVAR(shadowSetting)]) == 1) then {
+    if (_name in [QGVAR(colorTextMax), QGVAR(colorTextMin), QGVAR(colorShadowMax), QGVAR(colorShadowMin), QGVAR(textSize), QGVAR(shadowSetting)]) then {
```

Comment:
Still worked when I tested it, so case in conserved for name.
Also, the `setupTextColors` function gets called multiple times at init because the auto-convert adds  `ace_common_fnc_cbaSettings_settingChanged` to the script argument for `CBA_settings_fnc_init` and when all the settings get refreshed before `ace_settingsInitialized`, each triggers `setupTextColors`. The `ace_settingChanged` EH could be moved to inside the `ace_settingsInitialized` block to only have setup happen once at init?

Reply:
That should work.